### ISSUE TITLE
Sharethrough performance drops dramatically in Prebid v5.13.0

### ISSUE THREAD

**Issue by nanek** [prebid/Prebid.js#8003]
## Type of issue
bug

## Description
Sharethrough performance drops over 50% when upgrading to Prebid v5.13.0. Is there a Sharethrough technical contact that can look into this further? Thanks!
No other bidders are impacted.

## Steps to reproduce
Prebid 5.12.0 works fine.
Tested in Prebid 6.4.0 it did not perform well.
Isolated the the issue started in Prebid 5.13.0.

### Expected results

Upgrading Prebid to latest should not break integration with Sharethrough.

### Actual results

Sharethrough adapter not working correctly.

## Platform details
Prebid v5.13.0

**Comment by patmmccann** [comment]
This is fixed here https://github.com/prebid/Prebid.js/pull/7509 . Your Sharethrough account rep didn't tell you not to download this version?

**Comment by patmmccann** [comment]
duplicate of https://github.com/prebid/Prebid.js/issues/7510

**Comment by nanek** [comment]
Thanks for the response. We've been in contact with our Sharethrough rep since Nov 17. At that time we were testing upgrading from v4.41 to v6.1 and seeing a drop in Sharethrough performance of 50%. They indicated that it was fixed in v5.17, however v6.1 already contained those commits.  We then tested v6.4 and continued to see a 50% drop in Sharethrough performance from our baseline of v4.42. 

As part of our troubleshooting, I setup another test with v5.13 as it looked to introduce the issue. Sharethrough performance dropped 85%. It does seem like the fix you mentioned helped (only 50% drop instead of 85%), but didn't completely resolve the issue. In hindsight, it would have been more appropriate for me to title this issue as "Sharethrough performance still low after fix in https://github.com/prebid/Prebid.js/pull/7509" 

Any suggestions on how to proceed are greatly appreciated. Are there any Sharethrough contacts that monitor this repo that could assist us?
